doc/authoring_command_modules/authoring_commands.md

```diff
@@ -35,14 +35,15 @@ from azure.cli.commands import cli_command
 
 The signature of this method is 
 ```Python
-def cli_command(module_name, name, operation, client_factory=None, transform=None, table_transformer=None):
+def cli_command(module_name, name, operation, client_factory=None, transform=None, table_transformer=None, confirmation=None):
 ```
 You will generally only specify `name`, `operation` and possibly `table_transformer`.
   - `module_name` - The name of the module that is registering the command (e.g. `azure.cli.command_modules.vm.commands`). Typically this will be `__name__`.
   - `name` - String uniquely naming your command and placing it within the command hierachy. It will be the string that you would type at the command line, omitting `az` (ex: access your command at `az mypackage mycommand` using a name of `mypackage mycommand`).
   - `operation` - The handler that will be executed. Format is `<module_to_import>#<attribute_list>`
       - For example if `operation='azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.get'`, the CLI will import `azure.mgmt.compute.operations.virtual_machines_operations`, get the `VirtualMachinesOperations` attribute and then the `get` attribute of `VirtualMachinesOperations`.
   - `table_transformer` (optional) - Supply a callable that takes, transforms and returns a result for table output.
+  - `confirmation` (optional) - Supply True to enable default confirmation. Alternatively, supply a callable that takes the command arguments as a dict and returning a boolean. Alternatively, supply a string for the prompt.
```

Comment:
I'm not entirely sure I understand this. There are three cases:

1) True => default prompt enabled
2) callable => you have access to the command namespace (?) and return true or false? Do we use this?
3) string => enables the prompt with a custom message.

Does using #2 mean you essentially are using your own handler and throwing away the default one?